### ISSUE TITLE
Links with tooltips provide no additional information

### ISSUE THREAD

**Issue by mmiron-HC** [mmiron-HC/WST-RSAMS#69]
URL: most pages
User Agent: chrome
Language: both

**Issue Description:** 

- Links with tooltips ("title" attribute) do not provide additional information and  should be removed.
- Not all screen readers support the title attribute. However, those that do will read the link text and the title attribute one after the other making it redundant/confusing. 
- These tooltips are also only available to mouse users, keyboard users would never be presented with these tooltips.
- some of the tooltips don't seem to be related to the link.

![image](https://user-images.githubusercontent.com/65728700/129185140-36bfc2f5-3f79-47b5-b4ef-1291c58110ae.png)

**Steps to reproduce:**
Using mouse, hover over the links in the navigation. tooltip will appear.

**Expected results:**
Remove these tooltips. Best to avoid using tooltips.

**Comment by davidgotlieb** [comment]
Resolved. Tooltips disabled as recommended.

**Comment by qanitah** [comment]
They are still there - English and French

![image](https://user-images.githubusercontent.com/14280062/136237429-4c820491-4cff-4bc0-8a1d-9ec58c727cd9.png)

![image](https://user-images.githubusercontent.com/14280062/136237591-f30ce2e6-4ce7-4242-b1af-74379c211637.png)

**Comment by echouin** [comment]
From OP: "This is not relevant to public facing pages and it is not a critical or serious backend.﻿ Could you please approve the closure?"

**Comment by mmiron-HC** [comment]
@echouin as previously mentioned, we will only close issues once they are fixed.

**Comment by echouin** [comment]
@mmiron-HC This is a Drupal WxT issue. OP unable to fix at this time.

**Comment by echouin** [comment]
@mmiron-HC looks like this item is now fixed in Drupal Wxt: On branch 9.1.x the issue from this ticket is not occurring, think we can close this ticket.

**Comment by hydegeoff** [comment]
This issue is fixed on the Dashboard, but when going to edit profile, the tooltips come back.  The tooltip should removed on all pages related to Top Navigation bar.  As shown in the screenshot below, the tooltip is not being displayed correctly and "Edit Tags" is always in English no what your language settings are.
![English tooltip appears on French page](https://user-images.githubusercontent.com/88147013/160174989-2aa49a74-c91d-44c6-a555-4a3ae4387108.jpg)

**Comment by echouin** [comment]
@hydegeoff from OP:
![tooltip (2)](https://user-images.githubusercontent.com/88099454/160477324-179015b4-21f9-40b7-a525-600bf44531c6.png)

**Comment by echouin** [comment]
![tooltip (2)](https://user-images.githubusercontent.com/88099454/160477267-c4ccbc33-5734-4865-928c-3ae51a5549f7.png)

**Comment by hydegeoff** [comment]
@echouin, It is can be reproduced in Dev, as shown on the Edit Tag page below:  
![Edit Tags - Tooltips](https://user-images.githubusercontent.com/88147013/160616974-3adb2f68-fb2c-437a-8971-8db57a6dfa42.jpg)

**Comment by davidgotlieb** [comment]
@echouin I'm seeing the same think as @hydegeoff in Prod as well

**Comment by echouin** [comment]
@hydegeoff thank you for the added info, passed back to OP.

**Comment by echouin** [comment]
Not critical. Most likely a WxT issue. Might be resolved in a future update.
